src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -195,7 +195,10 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
           return search<
             IKibanaSearchRequest<ESQLSearchParams>,
             IKibanaSearchResponse<ESQLSearchReponse>
-          >({ params }, { abortSignal, strategy: ESQL_ASYNC_SEARCH_STRATEGY }).pipe(
+          >(
+            { params: { ...params, dropNullColumns: true } },
```

Comment:
Should it be configurable?
This will overwrite the value to `true` no matter what's set.

Reply:
I want it always true for expressions as I say in the description!